regulations/static/regulations/css/less/module/comment.less

```diff
@@ -316,16 +316,54 @@ Write Mode
     }
   }
 
+  .comment-context-wrapper {
+    border-radius: 2px;
+    box-shadow: 0 0 1px 1px @text_area_border;
+    margin: 30px 0;
+    position: relative;
+  }
+
+  .comment-context-toggle {
+    background-color: @grey;
+    border-radius: 2px;
+    cursor: pointer;
+    font-size: 16px;
+    padding: 10px 15px;
+
+    .comment-context-text {
+      color: @blue;
+    }
+
+    .fa-minus-circle,
+    .comment-context-text-less {
+      display: none;
+    }
+
+    .fa {
+      margin-right: 3px;
+    }
+
+    .comment-context-label {
+      font-weight: bold;
+    }
+
+    .comment-context-type {
+      text-transform: capitalize;
```

Comment:
Should it be "CFR" in this case? Then instead of transforming text in CSS, would a variable check with `{{ type|capfirst }}` or `{{ type|upper }}` be ok?